symengine/lambda_double.h

```diff
@@ -39,32 +49,39 @@ class LambdaDoubleVisitor : public BaseVisitor<LambdaDoubleVisitor<T>>
 
     T call(const std::vector<T> &vec)
     {
-        return result_(vec);
+        return result_(vec.data());
+    }
+
+    void call(T *outs, const T *inps)
+    {
+        for (unsigned i = 0; i < results.size(); ++i) {
+            outs[i] = results[i](inps);
```

Comment:
I will check whether it has any performance improvement. We can check for the keyword using CMake.